src/closedwindows/closedwindows-controller.js

```diff
@@ -0,0 +1,19 @@
+(function() {
+    'use strict';
+
+    class ClosedWindowsCtrl {
+        constructor(storeService) {
+            this.storeService = storeService;
+            this.numbers = [1, 2, 3];
+            this.refreshClosedWindows();
+        }
+        refreshClosedWindows() {
+            this.storeService.refreshStore();
+            this.closedWindows = this.storeService.getPreviousClosedWindows().sort((a, b) => b.closed - a.closed);
```

Comment:
I wonder whether the sorting should be performed in the HTML with angular. I think in general controllers are used for bindings, and the UI is responsible for knowing its own order. I think the native filter `orderBy` could be used to do it? I wonder what other people think though: @WPFerg ?